content/applications/inventory_and_mrp/inventory/warehouses_storage/reporting/aging.rst

```diff
@@ -1,98 +1,224 @@
-======================
-Inventory aging report
-======================
+=========================
+Stock valuation dashboard
+=========================
 
 .. |SVLs| replace:: :abbr:`SVLs (stock valuation layers)`
 
-The inventory aging report evaluates all items in stock, providing insights into potentially sunken
-purchase costs and delays in profitability.
+When a company has physical assets, such as inventory, they often want to know approximately how
+much has been spent on these goods, or how much they are worth at the moment. This process of
+assigning a monetary value to account for inventory is known as *stock valuation*.
 
-Create customized pivot tables to analyze product, operation types, month, or company breakdowns.
-This helps identify products in stock that are at risk of passing their expiration or viability
-dates, or instances of rot/decay for fast-expiring items.
+This value is often reported for accounting purposes. For instance, an insurance company may want to
+know the value of goods stored in a warehouse, in the event of a flood or fire.
 
-.. note::
+:doc:`Stock valuation <../../product_management/inventory_valuation/using_inventory_valuation>`
+typically utilizes one of two accounting systems:
+
+- **Perpetual**: The inventory is constantly (perpetually) being updated, and the value is
+  constantly changing.
+- **Periodic**: The inventory value is checked on an occasional (periodic) basis, and the value is
+  set at this occasional time.
+
+Using :ref:`tracked inventory <inventory/product_management/tracking-inventory>` in Odoo
+necessitates a *perpetual* inventory accounting system because of the need to know when and where
+inventory exists, and how much of it is available or forecasted. There are a few common :ref:`stock
+valuation methods <inventory/warehouses_storage/costing_methods>` used in Odoo: *standard price*,
+*average cost* (AVCO), and *first in, first out* (FIFO) accounting. It is important to know that the
+valuation method chosen for a product impacts the calculation of several fields in the stock
+valuation reports.
+
+Open the dashboard
+==================
+
+Odoo's *Stock Valuation* dashboard displays the financial value of all tracked inventory, according
+to each product's stock valuation method. This report can provide insights into potential issues in
+the supply chain, such as sunken purchase costs or delays in profitability. To access the dashboard,
+go to :menuselection:`Inventory app --> Reporting --> Valuation`.
+
+.. important::
    The *Reporting* menu in *Inventory* is only accessible to users with :doc:`admin access
    <../../../../general/users/access_rights>`.
```

Comment:
Since we are already updating this doc... please update the format here :)
```suggestion
   The :menuselection:`Reporting` menu in **Inventory** is only accessible to users with :doc:`admin
   access <../../../../general/users/access_rights>`.
```